python/py-astropy/Portfile

```diff
@@ -62,6 +62,15 @@ if {${name} ne ${subport}} {
         build.cmd  ${python.bin} setup.py --no-user-cfg --offline --no-git
         destroot.cmd  ${python.bin} setup.py --no-user-cfg --offline --no-git
     } else {
+        if {${python.version} < 37} {
+            epoch           1
```

Comment:
Incremnting `epoch` Is required because `py36-astropy @4.2` could be installed; the Python version error appeared when importing the module.